internal/backends/winit/glcontext.rs

```diff
@@ -140,37 +141,44 @@ impl OpenGLContext {
     ) -> (Self, Rc<winit::window::Window>) {
         #[cfg(not(target_arch = "wasm32"))]
         {
-            let (maybe_window, config) = crate::event_loop::with_window_target(|event_loop| {
-                glutin_winit::DisplayBuilder::new()
-                    .with_preference(glutin_winit::ApiPrefence::PreferEgl)
-                    .with_window_builder(Some(window_builder))
-                    .build(
-                        event_loop.event_loop_target(),
-                        glutin::config::ConfigTemplateBuilder::new(),
-                        |configs| {
-                            configs
-                                .reduce(|accum, config| {
-                                    let transparency_check =
-                                        config.supports_transparency().unwrap_or(false)
-                                            & !accum.supports_transparency().unwrap_or(false);
-
-                                    if transparency_check
-                                        || config.num_samples() > accum.num_samples()
-                                    {
-                                        config
-                                    } else {
-                                        accum
-                                    }
-                                })
-                                .unwrap()
-                        },
-                    )
-                    .unwrap()
+            let window = crate::event_loop::with_window_target(|event_loop| {
+                window_builder.build(event_loop.event_loop_target()).unwrap()
```

Comment:
I wonder if we really should `unwrap()` here or give a better, more actionable error message.
Is that not what people will see if there is a problem with the display or something?
Or is it the following unwrap.

(Ideally, we should have made our API so slint user would be able to catch that error)

Reply:
Good point, this something to consider for the API. So before this PR we unwrapped and still do after this, but I agree that we should consider passing this to the caller in the public API.